src/Video-detail/video-detail.css

```diff
@@ -0,0 +1,129 @@
+.video-container{
+    width: 100%;
+    height: 390px;
+    margin-top: 1rem;
+}
+.max-width-do-not-stretch{
+    max-width: 58.5rem;
+    width: 100%;
+    margin: 2rem auto;
+}
+.video-tutor-details .avatar{
+    margin: 0 1rem 0 0;
+}
+.video-tutor-details{
+    display: flex;
+    align-items: center;
+    padding: 0.5rem 1rem;
+}
+
+.filter-divider-line{
+    background-color: #dedddf;
+    border: 0.5px solid #dedddf;
+}
+.primary-divider-line{
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+    border: 0.5px solid var(--primary-color);
+}
+
+.stacked-list li{
+    border: none;
+}
+
+.playlist-container{
+    padding: 0.5rem 1rem 0.5rem 0;
+    background-color: var(--bg-light-pink);
+    width: fit-content;
+    max-width: calc(100vw - 1rem);
+    position: absolute;
+    top: 1.5rem;
+    left:0rem;
+    display: flex;
+    flex-direction: column;
+}
+
+.add-to-playlist-cta-container{
+    position: relative;
+}
+
+.playlist-container .btn{
+    margin-top: 0;
+    white-space: nowrap;
+}
+.stacked-list li.margin-0-list-item{
+    margin-bottom: 0;
+}
+
+.scroll-bar-styled{
+    overflow: auto;
+}
+
+.scroll-bar-styled::-webkit-scrollbar {
+    width: 0.25rem;
+}
+
+.scroll-bar-styled::-webkit-scrollbar-thumb {
+background-color: rgb(216, 215, 216);
+}
+
+.scroll-bar-styled::-webkit-scrollbar-track {
+background: #edebef;
+}
+.video-details-section .CTA-Container > .btn{
+    font-size: 1.5rem;
+}
+.add-to-playlist-cta-container > .btn {
+    font-size: 1.5rem;
+}
+.add-new-playlist-input{
+    padding: 1rem 0 1rem 1rem;
+}
+
+.notes-cta-container .btn{
+    margin: 0;
+    padding: 0.5rem;
+    font-size: 0.8rem;
+}
+
+.video-details{
+    width: 70%;
+    padding: 1rem 2rem 1rem 0rem;
+    box-sizing: border-box;
+}
+.notes-section{
+    width: 30%;
+    height: 390px;
```

Comment:
same here, to make it consistent with all the other properties, you could use rem instead of px.

Reply:
Done